### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by danpclements** [X3n0m0rph59/eruption#56] [General]
I just wanted to thank you for this, excellent job. I am enjoying the heck out of it and thought perhaps there was nothing to fulfill my need for LED and macro support on my Roccat devices. I was thrilled to find this, and it works even better than the default software on Windows, in my opinion. Thank you so much, keep up the good work!

**Comment by X3n0m0rph59** [comment]
Many thanks! I am very happy to hear this!
